### ISSUE TITLE
show stats for non taken pictures

### ISSUE THREAD

**Issue by echeloni2p** [mapillary/mapillary_issues#1071]
In the android app, after taking a sequence, some stats would be nice to see, e.g. x pictures taken, y pictures NOT saved, reasons:
- shaking
- tilt
- system error
- gps signal not good enough

Also a audio error for any picture NOT taken in automatic mode would be fine.

And, if a picture was not taken for any reason, take a picture as soon as possible again, do not wait the time between 2 pictures. (E.g. if time between 2 pictures is 10 sec, picture taken, wait 10 sec, GPS bad, no picture taken, and as soon as GPS is bakc again take a picture. Current behaviour is to wait 10 sec after last picture take-tryout).

thanks

**Comment by HolgerJeromin** [comment]
> take a picture as soon as possible again,

Yes, the same is valid for processing. Often my phone is able to process in 1.5 s, but sometimes it needs 2.5 (setting is every two seconds an image) . Right now the next image is taken after 4 seconds instead of 3 seconds. This is a difference at high speed and complex situations you want to cover.

**Comment by domdomegg** [comment]
As well as a 'x pictures taken', a 'x more pictures can be taken', by calculating average photo size of last 5 photos, then dividing remaining storage space for Mapillary (percent of it at which it is set to stop) by average size.

**Comment by ovivoda** [comment]
_mapillary_issues is no longer an active repository._

_Let’s continue discussing features in our forum! We will import this issue as topic to the <a href="http://forum.mapillary.com ">Mapillary Forum</a> in the coming weeks. We’ll notify here once it’s been done. See you there!_
